src/components/Registration/ProducerRegistration.js

```diff
@@ -0,0 +1,1202 @@
+import React, {
+  useMemo,
```

Comment:
Can we make all the text in the confirmation stage and throughout the entirety of our registration Work sans?